style.css

```diff
@@ -1,3 +1,9 @@
+body {
```

Comment:
- [ ] Kindly set the <code>font-family</code> to Poppins, as your previous reviewer already pointed out 🔆🔆 It is always a good idea to follow the design from Figma. It also would help you practice your attention to details skills, which is a highly sought-after skill by recruiters 😇

Reply:
Ya I do it but it give stylelint error the font family poplin that is why remove it ok I add it again.